osh/hub/scan/notify.py

```diff
@@ -264,9 +264,13 @@ def send_scan_notification(request, scan_id):
     # recipient setting
     recipient = get_recipient(scan.username)
 
-    # message setting
+    recipients = set()
```

Comment:
I do not like the identifiers differing from each other with the trailing `s` only.  Could we please use `recipient_list` or `recipient_set` for the new one?